code/index.html

```diff
@@ -1,37 +1,90 @@
-  </head>
-  <body>
-    <section class="cards">
-      <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
-        <p id="killerName"></p>
-        <p id="killerAge"></p>
-        <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
-        <p id="killerDescription"></p>
-      </article>
-      <article class="weapon-deck" id="weaponCard">
-        <p>WEAPON</p>
-        <p id="weaponName"></p>
-        <p id="weaponWeight"></p>
-      </article>
-      <article class="room-deck" id="roomCard">
-        <p>ROOM</p>
-        <p id="roomName"></p>
-      </article>
-    </section>
-
-    <h1 id="mystery"></h1>
-
-    <button>REVEAL THE CRIME</button>
-
-    <script src="clue.js"></script>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" />
+
+  <!-- favicon -->
+  <link rel="icon"
+    href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22> 🔎</text></svg>">
+
+  <!-- google fonts -->
+  <link href="https://fonts.googleapis.com/css2?family=Pacifico&display=swap" rel="stylesheet">
+
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;0,900;1,700;1,900&display=swap"
+    rel="stylesheet">
+
+  <title>Cluedo</title>
+</head>
+
+<body>
+
+  <header>
+    <h1>Cluedo</h1>
+
+    <div class="game-info-container">
+      <p class="game-info">Welcome to Cluedo!</p>
+      <p class="game-info">Select a killer, then a weapon and a room.</p>
+    </div>
+
+    <a href="#main" class="header-btn">Start the game</a>
+
+  </header>
+
+  <main id="main">
+    <div class="game-board">
+
+      <section class="cards">
+
+        <!-- *** KILLER *** -->
+        <article class="killer-deck" id="killerCard">
+          <div class="inner-card-style">
+            <h2 id="cardHeadingKiller">Killer</h2>
+            <p id="killerName"></p>
+            <img id="killerImage" src="assets/killer.png" height="200" />
+            <p id="killerOccupation"></p>
+          </div>
+        </article>
+
+        <!-- *** WEAPON *** -->
+        <article class="weapon-deck" id="weaponCard">
+          <div class="inner-card-style">
+            <h2 id="cardHeadingWeapon">Weapon</h2>
+            <p id="weaponName"></p>
+            <img id="weaponImage" src="assets/weapon.png" height="200" />
+            <p id="weaponWeight"></p>
+          </div>
+        </article>
+
+        <!-- *** ROOM *** -->
+        <article class="room-deck" id="roomCard">
+          <div class="inner-card-style">
+            <h2 id="cardHeadingRoom">Room</h2>
+            <p id="roomName"></p>
+            <img id="roomImage" src="assets/crime-scene.png" height="200" />
+            <p id="roomLocation"></p>
+          </div>
+        </article>
```

Comment:
Super clean.  ✨ 
Move height to CSS?
Naming: `cardHeadingRoom` => `roomCardHeading` to be consistent